src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx

```diff
@@ -52,6 +53,8 @@ const BottomSheetBackdropComponent = ({
 
   //#region callbacks
   const handleOnPress = useCallback(() => {
+    onPress?.()
```

Comment:
you need to add `onPress` to the `handleOnPress` dependencies list.

Reply:
@gorhom Hey gorhom, ooopss... you are right, I somehow missed it.